chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
     }
 
-    chrome_save() {
-        const id_k = this.id;
-        try {
-            chrome.storage.sync.set({ [id_k]: JSON.stringify(this.list) });
-        } catch (err) {
-            g2t_log("chrome_save: failed! Error: " + JSON.stringify(err));
-        }
-        return this;
+    const instance_k = args.obj[eblcMapID];
+    if (!args.fn in instance_k) {
```

Biome (1.9.4)

[error] The negation operator is used unsafely on the left side of this binary expression.

Unsafe fix: Wrap the expression with a parenthesis


(lint/suspicious/noUnsafeNegation)